WalletWasabi.Fluent/ViewModels/AddWallet/Create/ConfirmRecoveryWordsViewModel.cs

```diff
@@ -133,8 +133,10 @@ private void EnableAvailableWords(bool enable)
 
 	private async Task OnNextAsync()
 	{
-		var dialogCaption = "This is needed to open and to recover your wallet. Store it safely, it cannot be changed.";
-		var password = await Navigate().To().CreatePasswordDialog("Add Password", dialogCaption, enableEmpty: true).GetResultAsync();
+		var dialogCaption = "It's a recovery words extension for more security.\n" +
+			"You will need it to open and to recover your wallet.\n" +
+			"Store it safely, it cannot be reset if lost.";
```

Comment:
```suggestion
			"Your passphrase can't be reset if you forget it.";
```

Reply:
This implies that the passphrase should be memorized instead of written down and stored.

Reply:
That's fair, how about "Back up your passphrase safely, you cannot reset it."?